src/Store/StoreSections.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
```

Comment:
Hi, can you explain a little more what you mean? What are the disadvantages? Which bit isn't common?